package.json

```
@@ -69,6 +69,7 @@
     "@rollup/plugin-node-resolve": "^16.0.1",
     "@rollup/plugin-typescript": "12.3.0",
     "@tsconfig/node18": "18.2.6",
+    "@types/node": "25.2.3",
```

Biome (2.3.14)

[error] The key `@types/node` was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)